app/hotels/src/allHotels/searchForm/SearchForm.js

```diff
@@ -40,6 +40,8 @@ type Props = {
   onLocationChange: (location: string) => void,
 };
 
+export const SEARCH_MIN_WIDTH = 360;
```

Comment:
What is this magic number? It works nicely for every device? Why exactly 360?

Reply:
I put comment there.
It's about ~100 width for each input (to not wrap date into 2 rows) + padding (10x4). Should work for each device.

Reply:
Yes, see my comment above about Zeplin design (https://zpl.io/aML67Y7). You adjusted the design again so if you agreed on this with someone - I am not aware of that.